acceptance/acceptance_test.go

```diff
@@ -278,6 +283,19 @@ func testAcceptance(t *testing.T, when spec.G, it spec.S, packFixturesDir, packP
 				builderName = value
 			})
 
+			when("builder.toml is invalid", func() {
+				it("displays an error", func() {
+					h.CopyFile(t, filepath.Join(packFixturesDir, "invalid_builder.toml"), filepath.Join(tmpDir, "invalid_builder.toml"))
```

Comment:
This test is failing in our combo test suites because we wouldn't expect this behavior to exist in previous versions of pack. We can skip this test using `h.SkipIf()` and some code similar to https://github.com/buildpacks/pack/blob/007ab0cb7d61d8c999efee282c82953b56bb5b80/acceptance/acceptance_test.go#L536-L540.